public/app/core/services/util_srv.ts

```diff
@@ -36,7 +37,7 @@ export class UtilSrv {
       },
     };
 
-    const elem = React.createElement(AngularModalProxy, modalProps);
+    const elem = React.createElement(provideTheme(AngularModalProxy), modalProps);
```

Comment:
@dominik without `provideTheme()` wrapper modal will always be dark when you show it via event emitter.